frontend/composables/api/pillarsApi.ts

```diff
@@ -0,0 +1,100 @@
+﻿import type { LLMFeedback, Pillar, PillarDTO } from '~/types/pillars'
```

Comment:
it's also technically possible to have types auto-included, but they have been a bit tricky for me. what worked in the end is create the type definition files in the utils folder with the .d.ts ending, and remove the export keyword